src/display_buffer.cc

```diff
@@ -312,7 +317,16 @@ DisplayLine parse_display_line(StringView line)
                 auto closing = std::find(it+1, end, '}');
                 if (closing == end)
                     throw runtime_error("unclosed face definition");
-                face = get_face({it+1, closing});
+                if (*(it+1) == '`')
```

Comment:
Cant say I like this syntax, but to be faire I dont like the general display line syntax `{face}`...

I think this is a good occasion to revise it, I am not sure what would be better, but I'd like to get closer to something more well known... Unfortunately none of Markdown, asciidoc, restructuredText seems to provide a syntax for face/color selection, so the remaining reference I can think of is `LaTeX` but its not very satisfying either.

Any idea on what could be a nice syntax here, at least supporting changing face colors and referencing a built-in value, maybe more in the future as we'll probably want to get better formatting support for info boxes at some point.